src/main/java/spleetwaise/address/model/AddressBook.java

```diff
@@ -78,17 +82,6 @@ public Optional<Person> getPersonByPhone(Phone phone) {
         return Optional.of(filteredPersonList.get(0));
     }
 
```

Comment:
Removed this method due to cyclic dependency, which makes AddressBook depend on CommonModel, but CommonModel depends on AddressBook.